datafiles/templates/UserSignupReset/SignupRequest.html.st

```diff
@@ -15,6 +15,8 @@ $hackagePageHeader()$
 In particular you need an account to be able to upload or help maintain packages.
 
 <p>Using the form below you can register an account.
+The FirstnameLastname format of usernames is encouraged but not required.
```

Comment:
almost pointless to encourage something w/o any rationale and in the same sentence point out it's optional anyway... :-)